docker/Makefile

```diff
@@ -43,14 +43,22 @@ ifeq ($(IS_LATEST),true)
 	@docker tag puppet/r10k:$(version) puppet/r10k:latest
 endif
 
-test: prep
+test:
 	@bundle install --path .bundle/gems
 	@PUPPET_TEST_DOCKER_IMAGE=puppet/r10k:$(version) bundle exec rspec r10k/spec
 
-publish: prep
+publish:
```

Comment:
I think the only thing that needs to be depend on `prep` is the `build` target.

Reply:
no, test and publish also depend on the version, so if you're running those on their own those depend on prep too.

Reply:
Oh yeah `publish` does need it for the version, but `test` doesn't since that's running the built image that's provided (and doesn't reference version anywhere).

I'll put them back I guess.